scripts/publish-documentation/publish.sh

```diff
@@ -116,22 +119,38 @@ do
   git switch gh-pages
   git clean -fdx
 
+  # For some reason, recent usage of this script failed since `.config` folder
+  # was preventing from making a commit, as it was "untracked" change.
+  # Therefore, this folder is pre-emptively removed here, just in case.
+  # We don't need it in `gh-pages` branch anyway.
+  log "[$PWD] Removing ./config folder."
+  rm -rf "./config"
+
   for path in $(echo "$paths" | tr "," "\n")
   do
     # Extracts the module name from a path. A path should use the "/" as a file separator.
     module="${path##*\/}"
 
+    log "[$PWD] Creating directory 'dokka-reference/$module/v/$version'"
     mkdir -p "dokka-reference/$module/v/$version"
-    cp -r "../$module/build/docs/dokka/" "dokka-reference/$module/v/$version/"
+
+    # Previous versions of Dokka produced their output in `dokka` subfolder.
```

Comment:
I don't think we should document this change in the output directory. This description explains well what this PR is about, but there is no reason why the next engineer who looks at the script after the PR is merged may need the information about how it was before.